src/plugins/intel_gpu/docs/gpu_plugin_ops_enabling.md

```diff
@@ -54,64 +54,62 @@
             ...
             ```
 
-        * This unit test is built into `clDNN_unit_tests`. It is a gtest application.
+        * This unit test is built into `clDNN_unit_tests`. It is a `gtest` application.
             ```
             # Show list of test cases
             openvino/bin/intel64/Debug$ ./clDNN_unit_tests64 --gtest_list_tests
             # Run test
             openvino/bin/intel64/Debug$ ./clDNN_unit_tests64 --gtest_filter=scatter_elements_update_gpu_fp16.*
             ```
-        
-        * Test scope needs to be comprehensive, but not wasteful. These tests run for every PRs in CI. Let's save the planet.
-    
+
+        * Test scope needs to be comprehensive, but not wasteful. These tests run for every PR in CI. Let's save the planet.
```

Comment:
:)

Reply:
Do we keep it? Or shall we go for more formal version? :)

Reply:
Honestly, I like both, up to you

Reply:
Let's keep the original :)